modules/GenericDataParser/src/server/generic_data_parser.R

```diff
@@ -4257,7 +4256,7 @@ getProtocolEndpointData <- function(protocol) {
 
   #go through the protocol data to wrangle the endpoint data into the dataframe
   for (lsState in protocol$lsStates) {
-    if (lsState[['lsKind']] == "data column order") {
+    if (lsState$ignored != TRUE && lsState$deleted != TRUE && lsState[['lsKind']] == "data column order") {
```

Comment:
*outside PR scope*: Ignore ignored states

When testing, I found that if deleted an endpoint in the GUI, the warning about new endpoints did not re-appear as expected because this code was not skipping ignored states.